### ISSUE TITLE
add quest line to app error submit

### ISSUE THREAD

**Issue by toddmedema** [ExpeditionRPG/expedition#44]
+ api request validation and email template

**Comment by JTimNolan** [comment]
Is there an existing endpoint for error submitting?

**Comment by toddmedema** [comment]
Yep! Here's it in the app: https://github.com/ExpeditionRPG/expedition/blob/master/services/app/src/actions/Web.tsx#L151

And it in the API: https://github.com/ExpeditionRPG/expedition/blob/master/services/api/src/Handlers.ts#L240

You'll also need to extend the feedback DB model here: https://github.com/ExpeditionRPG/expedition/blob/master/services/api/src/models/Feedback.ts

This one will require running the API server locally, so I've gone ahead and emailed you the config.json file for that to work / so that you don't need to set up your own GCP.
